### DISCUSSION TITLE
irc 팀 회식 메뉴 추천

### DISCUSSION THREAD

**Discussion by joonseong11** [42Peer/irc#25] [Polls]
회식 메뉴 추천 도 가능해요!

예산 약 3만원

**Comment by Deanioche** [comment]
[숙성회를 숨도 못 쉴 정도로 먹고 온 집, 두께감 최소 베개급, 정말 어렵게 찾은 숙성회 전문점, 서울 가성비 횟집 [개포동 연스시]](https://www.youtube.com/watch?v=Hgf1k1mp6zo)

**Reply by joonseong11** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋ 연스시 맛있죠
